src/osdc/ObjectCacher.h

```diff
@@ -567,6 +567,7 @@ class ObjectCacher {
   void retry_waiting_reads();
 
  public:
+  bool CFG_block_writes_upfront(void) { return cfg_block_writes_upfront; }
```

Comment:
Minor nit, but I'd rather not differentiate things solely by case like this. Maybe name the variable `cfg_block_writes_upfront_`?

Reply:
Sure